driver-core/src/test/java/com/datastax/driver/core/PrimitiveTypeSamples.java

```diff
@@ -56,6 +56,8 @@
                 .put(DataType.cint(), Integer.MAX_VALUE)
                 .put(DataType.text(), "text")
                 .put(DataType.timestamp(), new Date(872835240000L))
+                .put(DataType.date(), -2147467577)
```

Comment:
This should be a Date instead of an integer right?   The code in UserTypesTest#testPrimitiveDatatypes:290 expects a Date.